src/libponyc/options/options.c

```diff
@@ -242,7 +300,23 @@ ponyc_opt_process_t ponyc_opt_process(opt_state_t* s, pass_opt_t* opt,
         usage();
         return EXIT_0;
       case OPT_DEBUG: opt->release = false; break;
-      case OPT_BUILDFLAG: define_build_flag(s->arg_val); break;
+      case OPT_BUILDFLAG:
+        if(is_openssl_flag(s->arg_val))
+        {
+          if(validate_openssl_flag(s->arg_val))
+          { // User wants to add an openssl_flag,
+            // remove any existing openssl_flags.
+            remove_build_flags(valid_openssl_flags);
```

Comment:
This line can be wrapped in an `#if defined(PONY_DEFAULT_OPENSSL)` since there wouldn't be a default openssl build flag to remove otherwise.

Reply:
I'm in of favor minimizing assumptions and not adding conditionals if they aren't needed. So I'd rather leave it as is.

Reply:
Okey dokey. Works for me.